### ISSUE TITLE
Tiny documentation error with explanation of beta parameter

### ISSUE THREAD

**Issue by rsimmons** [sublee/trueskill#11]
The documentation describes beta as "the distance which guarantees about 75.6% chance of winning". I think the correct percentage should be 76.025% (rounded however you wish). While the difference is trivial, it might confuse other people.

I was curious where the 75.6 magic number came from so derived what it should be, using the formula for computing win probability (mentioned in another issue). If you consider a match of two players, with the player sigmas and draw margins being 0, and the difference in rating means equal to beta, the win probability simplifies to cdf(1/sqrt(2)), which is about 0.76025.

**Comment by sublee** [comment]
Thanks to report.  I agree on your opinion.

Should we describe Beta as `76.025% (Φ(1/√2))`?  Or `approximately 80%`?  Which do you prefer?

**Comment by sublee** [comment]
Which win probability formula you used?  If you used [the last one](https://github.com/sublee/trueskill/issues/1#issuecomment-149762508), should the simple formula be `Φ(1/√(2β))` instead of `Φ(1/√2)`?

```
Φ(delta of μ / √(players×β²＋sum of σ))
→ Φ(β / √(2β²＋0))
→ Φ(1 / √(2β))
```

**Comment by rsimmons** [comment]
I think there's a mistake in that simplification: `Φ(β÷√(2β²＋0))` has `√(2β²)` in the inside denominator, which simplifies to `β√2`, so we get  `Φ(β÷(β√2)) → Φ(1/√2)`.

I would probably describe beta as "the distance which guarantees about 76% (specifically Φ(1/√2)) chance of winning".

Also, thanks so much for your work on this project. When I discovered the TrueSkill site/papers I was excited to use it but the thought of implementing it from scratch made my heart sink a little :)

**Comment by sublee** [comment]
Oh, you're right.  I had a mistake.  Thank you for letting me know.

Would you give me a pull request to add you to the contributor list?  If you don't care about the contributor list, I'll update the document myself.

**Comment by sublee** [comment]
I released TrueSkill-0.4.4 with your patch.  http://trueskill.org/#version-0-4-4
